packages/dashboard/src/components/common/Button.tsx

```diff
@@ -1,14 +1,19 @@
 import type { MouseEventHandler } from "react";
 
 interface Props {
+  disabled: boolean;
```

Comment:
Maybe this needs to be optional:
```suggestion
  disabled?: boolean;
```